README.md

```diff
@@ -268,6 +268,21 @@ $ remme node-account transfer-tokens-from-frozen-to-unfrozen
 }
 ```
 
+Transfer available tokens from unfrozen reputational balance to operational balance (executable only on the machine which runs the node) — `remme node-account transfer-tokens-from-unfrozen-to-operational --amount=1000`.
```

Comment:
Why `available`? 

When you transfer tokens from frozen to operational, the system chooses the amount on own (you cannot specify it).

When you transfer tokens from unfrozen to operational, you specify the amount. So remove `available` from the description. Check all occurrence, please.

Reply:
Done.